airflow/providers/microsoft/azure/log/wasb_task_handler.py

```diff
@@ -92,7 +93,7 @@ def hook(self):
             )
             return None
 
-    def set_context(self, ti) -> None:
+    def set_context(self, ti: TaskInstance, **kwargs) -> None:
```

Comment:
doesn't this need the actual param `identifier`?

Reply:
same as https://github.com/apache/airflow/pull/32646/files#r1275121766